docs/src/app/components/pages/components/DropDownMenu/README.md

```diff
@@ -0,0 +1,6 @@
+## Drop Down Menu
+
+To find out more about the `DropDownMenu` component please visit the Material Design's
+specifications **WHERE IS IT IN THE SPEC?!**.
```

Comment:
Indeed, the spec isn't breaking thing too much into component.
But I'm wondering, what's the difference between the DropDownMenu and the SelectField?
I may be wrong, but I feel like the SelectField is enought.